From 3dc1a125c24af5a369a8a2a19f8764692ce5b926 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <grinberg@compulab.co.il>
Date: Mon, 17 Dec 2012 17:38:24 +0200
Subject: ARM: imx6: close the comment

Apparently, the original author has forgot to close the comment (*\).
Close it now.

Signed-off-by: Igor Grinberg <grinberg@compulab.co.il>
---
 arch/arm/mach-mx6/mx6_ddr_freq.S |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/mx6_ddr_freq.S b/arch/arm/mach-mx6/mx6_ddr_freq.S
index 8d649f3..1aedf2c8 100644
--- a/arch/arm/mach-mx6/mx6_ddr_freq.S
+++ b/arch/arm/mach-mx6/mx6_ddr_freq.S
@@ -550,7 +550,7 @@ poll_dvfs_clear_1:
 	cmp     r0, #0x2000000
 	beq     poll_dvfs_clear_1
 
-	/* if DLL was previously on, continue DLL off routine
+	/* if DLL was previously on, continue DLL off routine */
 	cmp    	r9, #1
 	beq 	continue_dll_off_3
 
-- 
1.7.9.5

